[2025-04-04T19:17:28.922+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-04T19:17:28.945+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tmdb_pipeline_v2.train_ml_model manual__2025-04-04T19:16:57.058244+00:00 [queued]>
[2025-04-04T19:17:28.956+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tmdb_pipeline_v2.train_ml_model manual__2025-04-04T19:16:57.058244+00:00 [queued]>
[2025-04-04T19:17:28.957+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 3
[2025-04-04T19:17:28.971+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): train_ml_model> on 2025-04-04 19:16:57.058244+00:00
[2025-04-04T19:17:28.977+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=5307) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-04T19:17:28.979+0000] {standard_task_runner.py:63} INFO - Started process 5309 to run task
[2025-04-04T19:17:28.979+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'tmdb_pipeline_v2', 'train_ml_model', 'manual__2025-04-04T19:16:57.058244+00:00', '--job-id', '140', '--raw', '--subdir', 'DAGS_FOLDER/dag_tmdb.py', '--cfg-path', '/tmp/tmpq41l21p8']
[2025-04-04T19:17:28.981+0000] {standard_task_runner.py:91} INFO - Job 140: Subtask train_ml_model
[2025-04-04T19:17:29.027+0000] {task_command.py:426} INFO - Running <TaskInstance: tmdb_pipeline_v2.train_ml_model manual__2025-04-04T19:16:57.058244+00:00 [running]> on host 1b0019c2839f
[2025-04-04T19:17:29.119+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gabriela' AIRFLOW_CTX_DAG_ID='tmdb_pipeline_v2' AIRFLOW_CTX_TASK_ID='train_ml_model' AIRFLOW_CTX_EXECUTION_DATE='2025-04-04T19:16:57.058244+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-04T19:16:57.058244+00:00'
[2025-04-04T19:17:29.121+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-04T19:17:30.537+0000] {warnings.py:110} WARNING - /opt/***/dags/dag_tmdb.py:820: UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
  df = pd.read_sql(query, conn)

[2025-04-04T19:17:30.986+0000] {dag_tmdb.py:899} INFO - Modelo ML entrenado y guardado en /opt/***/data/movie_analytics/ml_model_v2_20250404_191730
[2025-04-04T19:17:30.986+0000] {dag_tmdb.py:900} INFO - Métricas del modelo - MSE: 20112.9701, R²: -1.1902
[2025-04-04T19:17:31.043+0000] {python.py:237} INFO - Done. Returned value was: /opt/***/data/movie_analytics/ml_model_v2_20250404_191730
[2025-04-04T19:17:31.045+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-04T19:17:31.075+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=tmdb_pipeline_v2, task_id=train_ml_model, execution_date=20250404T191657, start_date=20250404T191728, end_date=20250404T191731
[2025-04-04T19:17:31.124+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-04T19:17:31.143+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-04T19:17:31.145+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
